### PR TITLE
Feature 54030 routes clean up

### PR DESCRIPTION
**Pull request by LBruegger**

### CODE REVIEW

File: frontend/translations/en.yml [hunk 1]
```diff
@@ -208,6 +208,8 @@ en:
       team: Team
     new:
       title: Create a new team
+    edit:
+      title: Edit an existing team
```
Comment (lucashabersaat) [inline frontend/translations/en.yml:212]: Will give you a merge conflict. Note to take over the master's version.

File: spec/system/account_modal_system_spec.rb [hunk 2]
```diff
@@ -57,13 +57,13 @@
...
     team = Team.find(group.team_id)
-    visit("/accounts/#{account.id}")
+    visit(account_path(account.id))
 
     expect(page).to have_link(id: 'edit_account_button')
     click_link(id: 'edit_account_button')
 
     expect(find('.modal-content')).to be_present
-    expect(page).to have_text('Edit Account')
+    expect(page).to have_text('Editing account')
```
Comment (lucashabersaat) [inline spec/system/account_modal_system_spec.rb:66]: Take over from master when merging.

File: config/routes/admin.rb [hunk 3]
```diff
@@ -5,18 +5,18 @@
...
 
-    resources :users, except: :destroy do
+    resources :users, except: [:destroy, :show] do
       member do
         get 'unlock'
       end
     end
 
-    resources :recryptrequests do
+    resources :recryptrequests, only: [:index, :destroy, :resetpassword] do
```
Comment (lucashabersaat) [inline config/routes/admin.rb:19]: Burn those dangling routes. :fire:

File: config/routes/api.rb [hunk 4]
```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  scope '/api', module: 'api' do
+  scope '/api', module: 'api', as: 'api' do
```
Comment (lucashabersaat) [inline config/routes/api.rb:4]: :muscle:

File: config/routes/admin.rb [hunk 5]
```diff
@@ -5,18 +5,18 @@
     resources :maintenance_tasks, only: :index
     post '/maintenance_tasks/:id/execute', to: 'maintenance_tasks#execute', as: 'maintenance_tasks_execute'
 
-    resource :settings do
+    resource :settings, only: [:index, :update_all] do
```
Comment (psunix) [inline config/routes/admin.rb:8]: remove update_all from only. it's not a standard crud action

File: config/routes/admin.rb (hunk 3 above)
Comment (psunix) [reply config/routes/admin.rb:19]: remove resetpassword from only. same as comment before ....

File: config/routes/api.rb (hunk 4 above)
Comment (psunix) [reply config/routes/api.rb:4]: :fireworks:

**Pull request merged**